### ISSUE TITLE
consumer offset

### ISSUE THREAD

**Issue by VirginDaddy** [yahoo/kafka-manager#335]
I am using kafkaspout as consumer, which writes consumers offset to zookeeper under the defined zkRoot. I am not sure if I can pass explicit zkRoot to search for consumers.Currently I guess kafka-manager looks for consumers from zookeeper under consumers directory so its not showing up in UI.

**Comment by songxin1990** [comment]
yes,before 0.8,Consumer Tab only show consumers in zookeeper.Consumer information stored in Kafka(where is in __consumer_offsets topic) is visible.But now u can use kafka-manager version after 0.9 
to solve this problem.
